build-zip.sh

```diff
@@ -12,7 +12,8 @@ cp composer.lock build/organic/
 cp -r vendor build/organic/
 (cd affiliate/ && npm install && npm run build)
 mkdir -p build/organic/affiliate/
-cp -r affiliate/build build/organic/affiliate/
+cp -r affiliate/blocks/productCard/build build/organic/affiliate/productCard
+cp -r affiliate/blocks/productCarousel/build build/organic/affiliate/productCarousel
 cd build
 zip -r organic-${BUILD_NUMBER}.zip organic
 cp organic-${BUILD_NUMBER}.zip organic.zip
```

Comment:
I'm not sure how this build file should look. Winging it while I read up on it some more

Reply:
This looks reasonable, but it's maybe something @sergunich has better insights into.

Reply:
Maybe @regular-alves could take a quick look at this build file. Amy mentioned that you might be able to take a quick look and confirm.

Reply:
Thanks for taking a look!

Reply:
I think this is actually pretty easy. We just need to copy the existing directory structure into build/organic. In other words, https://github.com/orgnc/wordpress-plugin contains `organic.php`, `Organic`, `affiliate`, etc., so `build/organic` should contain `organic.php`, `Organic`, `affiliate`, etc. `affiliate` itself contains `blocks/productCard` and `blocks/productCarousel` and `blocks/shared`, so these should be copied appropriately (so that, when this zip is uploaded into WordPress, the register_block_type path we've defined--based on blocks/...--works). So I think what we want is:

```
(cd affiliate/ && npm install && npm run build)
mkdir -p build/organic/affiliate/blocks/
cp -r affiliate/blocks/productCard/build build/organic/affiliate/blocks/productCard/
cp -r affiliate/blocks/productCarousel/build build/organic/affiliate/blocks/productCarousel/
cp -r affiliate/blocks/shared build/organic/affiliate/blocks/shared/
cd build
zip -r organic-${BUILD_NUMBER}.zip organic
cp organic-${BUILD_NUMBER}.zip organic.zip
```

That said, I'm still trying to find a good way to test.

Reply:
Yeah, makes sense

Reply:
My above was the right idea, but executed incorrectly. Here is what the code should be:

```
(cd affiliate/ && npm install && npm run build)
mkdir -p build/organic/affiliate/blocks/
cp -r affiliate/blocks/ build/organic/affiliate/
cd build
zip -r organic-${BUILD_NUMBER}.zip organic
cp organic-${BUILD_NUMBER}.zip organic.zip
```

This will recreate the directory structure correctly.

It turns out this is super trivial to test once you understand how it all works: delete wordpress-plugin/build (if applicable); run the build script in the container; replace the mu-plugins/wordpress-plugin directory with the newly built wordpress-plugin/build/organic directory. E.g., I moved wordpress-plugin to my Desktop temporarily, then moved its build/organic subdirectory into the mu-plugins directory and renamed it to wordpress-plugin. Everything should still work in the WP editor--after all, you've just replaced your working wordpress-plugin directory with what should be another working wordpress-plugin directory. I've confirmed that the above code works and I've confirmed that this testing procedure works.

Once this change is made, we can finally merge this PR. I'll update documentation because this is something that could easily cost others development time when it really shouldn't.

Reply:
Awesome! Thanks for testing it

Reply:
Confirming that the above works. I don't know why it wouldn't, but I've been staring at this so long that nothing makes sense, so good to double-check.

Reply:
Also, @amittman, be sure to update the wordpress-plugin version in solutions-wordpress.

Reply:
Yep, was waiting until this was ready to go